compose/bin/create-user

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env bash
+
+# Function to create admin user
+create_admin_user() {
+  USERNAME="admin"
+  EMAIL="admin@example.com"
+  FIRSTNAME="Admin"
+  LASTNAME="Admin"
+  PASSWORD="Admin123"
+
+  bin/magento admin:user:create \
+     --admin-user="${USERNAME}" \
+     --admin-password="${PASSWORD}" \
+     --admin-email="${EMAIL}" \
+     --admin-firstname="${FIRSTNAME}" \
+     --admin-lastname="${LASTNAME}"
+
+  if [ $? -eq 0 ]; then
+      echo "Admin account created successfully."
+      echo "Username: ${USERNAME}"
+      echo "Email: ${EMAIL}"
+      echo "Firstname: ${FIRSTNAME}"
+      echo "Lastname: ${LASTNAME}"
+      echo "Password: ${PASSWORD}"
+  else
+      echo "Error creating admin user."
+  fi
+}
+
+# Function to create customer account
+create_customer_account() {
+  EMAIL="${EMAIL:-customer@example.com}"
+  PASSWORD="${PASSWORD:-customer123QWERTY}"
+  FIRSTNAME="${FIRSTNAME:-Customer}"
+  LASTNAME="${LASTNAME:-Customer}"
+  WEBSITE="${WEBSITE:-1}"
+
+  bin/n98-magerun2 customer:create "${EMAIL}" "${PASSWORD}" "${FIRSTNAME}" "${LASTNAME}" "${WEBSITE}"
+
+  if [ $? -eq 0 ]; then
+      echo "Customer account created successfully."
+      echo "Email: ${EMAIL}"
+      echo "Firstname: ${FIRSTNAME}"
+      echo "Lastname: ${LASTNAME}"
+      echo "Password: ${PASSWORD}"
+  else
+      echo "Error creating customer account."
+  fi
+}
+
+# Check if n98-magerun2.phar exists, if not download and install
```

Comment:
We don't need lines 51-57 as they are already covered in the `bin/n98-magerun2` script https://github.com/markshust/docker-magento/blob/master/compose/bin/n98-magerun2